sdk/go/common/util/executable/executable.go

```diff
@@ -32,18 +32,27 @@ func FindExecutable(program string) (string, error) {
 		return cwdProgram, nil
 	}
 
-	// look in $GOPATH/bin
+	// look in potentials $GOPATH/bin
 	if goPath := os.Getenv("GOPATH"); len(goPath) > 0 {
-		goPathProgram := filepath.Join(goPath, "bin", program)
-		fileInfo, err := os.Stat(goPathProgram)
-		if err != nil {
-			if !os.IsNotExist(err) {
-				return "", errors.Wrapf(err, "unable to find program in %q", goPathProgram)
+		// getPotentialPaths will return paths where to look.
+		// Because the GOPATH can take the form of multiple paths (e.g: GOPATH="home/user/go:/usr/local/go")
+		// we need to split the GOPATH, and look into each of the paths.
+		// If the GOPATH hold only one path, there will only be one element in the slice.
+		potentialPaths := getPotentialPaths(goPath, runtime.GOOS)
```

Comment:
Here instead of having a function that takes no parameters, I preferred to add these two, so it's more testable.